docs/e2e-testing.md

```diff
@@ -14,6 +14,16 @@ A test app, test library and test cases are in [`/packages/e2e-test-app/`](../pa
 
 This will be automatically done for you if you use the [RNW dependency script](https://microsoft.github.io/react-native-windows/docs/rnw-dependencies) with `rnwDev` as an argument.
 
+The E2E tests assume it is installed in the default location under `C:\Program Files (x86)\Windows Application Driver\WinAppDriver.exe`. If you choose a different path, you will need to specify it in [`/packages/e2e-test-app/jest.config.js`](../packages/e2e-test-app/jest.config.js).
```

Comment:
`%ProgramFiles(x86)%` ?

Reply:
I think the code I copied from wdio-winappdriver-service hardcodes the above. Using the env variable would be better though.